Active Record Multiple Database > Granular Database Connection Switching (#695)
campuscode/rails-guides-pt-BR: pt-BR/active_record_multiple_databases.md

end
```

## Alternando Conexão de Banco de Dados Granular

No Rails 6.1 é possível alternar conexões para um banco de dados ao invés de
todos os bancos de dados globalmente. Para usar este recurso, você deve primeiro definir

> `config.active_record.legacy_connection_handling` para` false` nas configurações do seu
> aplicativo. A maioria dos aplicativos não precisam fazer nenhuma outra

Reviewer: ```suggestion
`config.active_record.legacy_connection_handling` para` false` nas configurações da sua
aplicação. A maioria das aplicações não precisam fazer nenhuma outra
```

> ## Granular Database Connection Switching

Reviewer: ```suggestion
## Alternando Conexão de Banco de Dados Granular
```
Acredito ser melhor já traduzir o título também. O que acha? :smile:

Author: sim, obrigado por apontar

> No Rails 6.1 é possível alternar conexões para um banco de dados ao invés de
> todos os bancos de dados globalmente. Para usar este recurso, você deve primeiro definir
> `config.active_record.legacy_connection_handling` para` false` em seu aplicativo
> configuração. A maioria dos aplicativos não precisa fazer nenhum outro
> mudanças, uma vez que as APIs públicas têm o mesmo comportamento.

Reviewer: ```suggestion
No Rails 6.1 é possível alternar conexões para um banco de dados ao invés de
todos os bancos de dados globalmente. Para usar este recurso, você deve primeiro definir
`config.active_record.legacy_connection_handling` para` false` nas configurações do seu
aplicativo. A maioria dos aplicativos não precisam fazer nenhuma outra
alteração, uma vez que as APIs públicas têm o mesmo comportamento.
```

> Com `legacy_connection_handling` definido como false, qualquer classe de conexão abstrata

Reviewer: ```suggestion
Com `legacy_connection_handling` definido como _false_, qualquer classe de conexão abstrata
```

end
```

> Para mudar apenas o cluster de banco de dados primário, use `ApplicationRecord`:

Reviewer: ```suggestion
Para mudar apenas o [_cluster_](https://pt.wikipedia.org/wiki/Cluster) de banco de dados primário, use `ApplicationRecord`:
```

Acredito que podemos colocar uma referência de para _cluster_ aqui, o que acha?

Author: sim, faz sentido

end
```

> `ActiveRecord :: Base.connected_to` mantém a capacidade de alternar

Reviewer: ```suggestion
`ActiveRecord::Base.connected_to` mantém a capacidade de alternar
```
